content/reference/endpoints/fees.md

```diff
@@ -11,41 +11,43 @@
-currency          | String  | Read/Write | Requested fee's currency, defaults to `original_currency`. List of currencies codes described in [enums section](/reference/enums#currencies)
+Name                   | Type    | Read/Write | Description
+-----------------------|---------|------------|------------
+id                     | Integer | Read       | Fee's id.
+-----------------------|---------|------------|------------
+currency               | String  | Read/Write | Requested fee's currency, defaults to `original_currency`. List of currencies codes described in [enums section](/reference/enums#currencies)
 downpayment_percentage | [Decimal](/reference/enums#formats) | Read/Write |  ** Required** Fee's downpayment percentage.
-exchange_rate     | [Decimal](/reference/enums#formats) | Read       | The rate used to convert from the `original_currency` to `currency`.
-name              | [Object](/reference/enums#formats)   | Read/Write | ** Required** Fee's name, list of locales described in [enums section](/reference/enums#locales).
-kind              | String  | Read/Write | Fee's kind, list of fee's kinds described in [enums section](/reference/enums#fee-kinds)
-original_currency | String  | Read       | Fee's original currency, list of currencies codes described in [enums section](/reference/enums#currencies)
-rate              | [Decimal](/reference/enums#formats)  | Read/Write |  ** Required** Fee's rate.
-rate_kind         | String  | Read/Write |  ** Required** Fee's rate kind, list of fee's rate kinds described in [enums section](/reference/enums#fee-rate-kinds).
-------------------|---------|------------|------------
-archived_at       | [Time](/reference/enums#formats) | Read         | Fee's archive time.
-created_at        | [Time](/reference/enums#formats) | Read         | Fee's create time.
-updated_at        | [Time](/reference/enums#formats) | Read         | Fee's update time.
+exchange_rate          | [Decimal](/reference/enums#formats) | Read       | The rate used to convert from the `original_currency` to `currency`.
+name                   | [Object](/reference/enums#formats)   | Read/Write | ** Required** Fee's name, list of locales described in [enums section](/reference/enums#locales).
+kind                   | String  | Read/Write | Fee's kind, list of fee's kinds described in [enums section](/reference/enums#fee-kinds)
+original_currency      | String  | Read       | Fee's original currency, list of currencies codes described in [enums section](/reference/enums#currencies)
+rate                   | [Decimal](/reference/enums#formats)  | Read/Write |  ** Required** Fee's rate.
+rate_kind              | String  | Read/Write |  ** Required** Fee's rate kind, list of fee's rate kinds described in [enums section](/reference/enums#fee-rate-kinds).
+maximum_possible_rate  | [Decimal](/reference/enums#formats) | Read/Write | Fee's maximum possible rate, calculated value won't exceed amount present on this attribute.
```

Comment:
this is used only for inidivual percentage per adult, no? good to mention that

Reply:
not really. It's used for all "percentage" attributes so I'll add info about it